File: 프로그래머스/lv0/120902. 문자열 계산하기/문자열 계산하기.js
function solution(my_string) {
  var answer = 0;
  const result = my_string.split(' ');
  let i = 0;
  while (true) {
    if (result[i] === '+') {
      answer += Number(result[i+1])
      i += 2;
    } else if (result[i] === '-') {
      answer -= Number(result[i+1])
      i += 2;
    } else {
      answer = Number(result[i])
      i += 1;
    }
    if (i >= result.length) {
      break
    }
  }

  return answer;
}

File: 프로그래머스/lv0/120902. 문자열 계산하기/문자열 계산하기.py
def solution(my_string):
    my_string = my_string.split()
    answer = int(my_string.pop(0))
    stack = []
    
    for cmd in my_string:
        if not cmd.isdigit():
            stack.append(cmd)
        else:
            check = stack.pop()
            if check == "+":
                answer += int(cmd)
            else:
                answer -= int(cmd)
        
    return answer